src/behavior-property-observer.js

```diff
@@ -38,7 +38,7 @@ export class BehaviorPropertyObserver {
   setValue(newValue: any): void {
     let oldValue = this.currentValue;
 
-    if (oldValue !== newValue) {
+    if (Object.is(newValue, oldValue)) {
```

Comment:
@bigopon Should there be a `!` in front of this?

Reply:
Yes, it's fixed. Thanks for catching that.